### ISSUE TITLE
Semantic and Linked Data representation

### ISSUE THREAD

**Issue by wschella** [ZeusWPI/zeus.ugent.be#175]
It would be very cool to semantically describe our structured data and make it available to be used by other projects. Most of the data we have (ex: events, projects, blogs) is very structured already, so this would come down to defining a clear mapping to linked data concepts and making it available in the right format.

**Comment by werthen** [comment]
@RubenVerborgh would love this 😉

**Comment by RubenVerborgh** [comment]
You're right, I do!

However, @wschella, if there's one thing you should've learned from lab 3 of Web development, it's that you should _not_ define your own ontology, but instead try to reuse first ;-) Especially schema.org seems to cover a lot of ground. Let me know how I can help.

Also, after you've semantically marked up concepts, it's straightforward to make your data queryable. I've [done that](http://query.verborgh.org/) with [my data](https://data.verborgh.org/ruben) and [published the process](https://ruben.verborgh.org/articles/queryable-research-data/).

**Comment by werthen** [comment]
#171 is related to this issue, as we need to have all that event data

**Comment by wschella** [comment]
@werthen agreed, but this could happen retroactively, and as such is not a dependency for this issue. 
Using existing ontologies will indeed be beneficial to get this up and running correctly, however, considering that writing your own ontology might be half the fun, I have the following question @RubenVerborgh: using an ETL pipeline as you described in your publication, which is equipped with reasoning, how much does it matter what ontologies you use initially? As I understand, the defined ETL process would expose triples in predefined ontologies, as long as they are linked correctly.

**Extra:** 
A big downside for a custom vocabulary would be that the RDFa itself is meaningless for crawlers (and all other non-inhouse agents for that matter), benefits might include being able to dynamically update the mappings, and a consistent (and newbie friendly) namespace across the site. This last benefit could also be achieved by using schema.org for _everything_.

**Comment by werthen** [comment]
Agreed, could happen retroactively, but I'd like to get it right from the very first moment

**Comment by RubenVerborgh** [comment]
> using an ETL pipeline as you described in your publication, which is equipped with reasoning, how much does it matter what ontologies you use initially?

As you say:

> A big downside for a custom vocabulary would be that the RDFa itself is meaningless for crawlers

So what I do is include the most widely understood vocabulary in the HTML, and have the rest entailed by the reasoner for publication in the TPF interface.
